docs/customization/authorizations.md

```diff
@@ -0,0 +1,29 @@
+# Authorizations
+
+One particular thing about this kind of applications is the need to Authorize a given user. This is specially important when you want to have legally bindings decisions taken on the platform. There are several ways that this could be done:
+
+* By sending a SMS code to users to verify that their have a valid cellphone
+
+* By allowing users to upload a photo or scanned image of their identity document
+
+* By sending users a code through postal code
+
+* By allowing users to go to to a physical office and check their documentation
+
+* By checking some information through other systems (as a Municipal Census on the case of Municipalities, Cities or Towns)
+
+* By having a list of valid users emails
+
+Right now Decidim supports only a few of these cases, but we have an internal API where you can program your own kind of verifications. For creating your own AuthorizationHandler for an external API (ie a Municipal Census) you should add a app/services/ file, then activate it on config/initializers/decidim.rb and finally enabling it on /system for the tenant.
```

Comment:
Here you're using "verifications", should we unify the language?

Also, "For creating your own" => "To create your own"